extension/parquet/column_writer.cpp

```diff
@@ -330,6 +331,7 @@ class BasicColumnWriter : public ColumnWriter {
 	//! We limit the uncompressed page size to 100MB
 	// The max size in Parquet is 2GB, but we choose a more conservative limit
 	static constexpr const idx_t MAX_UNCOMPRESSED_PAGE_SIZE = 100000000;
+	static constexpr const idx_t MAX_UNCOMPRESSED_DICT_PAGE_SIZE = 10 * MAX_UNCOMPRESSED_PAGE_SIZE;
```

Comment:
What is the a reason for this to deviate from the 100MB limit? Perhaps hardcode it to 1GB instead, and explain why this deviates in a comment above?